### ISSUE TITLE
Generate unique identifier for catalogued object

### ISSUE THREAD

**Issue by cgendreau** [DINA-Web/dina-model-concepts#27]
`CatalogNumber` are not unique. We need to define the scope of uniqueness: collection, institution or global.

Global would imply DOIs and it may not be available for all institutions.

**Comment by cgendreau** [comment]
Yes it has 2 different needs: capture existing catalogNumber + generate one. I will create a PR with definitions of `catalogNumber `and `occurrenceID `based on what you wrote.

**Comment by cgendreau** [comment]
Suggested additions to the glossary: https://github.com/DINA-Web/collection-specs/pull/28

**Comment by cboelling** [comment]
Our strategy at the Museum für Naturkunde is to use globally unique URIs as primary identifiers for any resource (including catalogued objects). These primary IDs are minted at our institution and CETAF-conformant. They are intuitive enough to also serve as catalogue numbers used by the collections.
Any other ID (catalogue numbers, inventory numbers, etc.) in local use (legacy or new) is connected as an additional object datum to that primary  identifier.
Regarding the data model this means that any resource has a primary identifier of some kind (preferably globally unique) and can have arbitrary many additional identifiers of various types, for which no uniqueness assumptions are made.

[1] https://cetaf.org/cetaf-stable-identifiers
